src/observable/throw.ts

```diff
@@ -4,7 +4,7 @@ import {Subscription} from '../Subscription';
 
 export class ErrorObservable<T> extends Observable<T> {
 
-  static create<T>(error: T, scheduler?: Scheduler) {
+  static create<T>(error: any, scheduler?: Scheduler) {
```

Comment:
For functionality it's not needed, but for type inference by TypeScript it may sometimes be needed.
For example, you need to assign it to an object of type `Observable<Item>` and there is not chance for TypeScript to infer that type by the error of type `any`, and `ErrorObservable<any>` is not assignable to `Observable<T>`, then you need to provider it by `<Item>` mannualy.